client/src/components/FormInput/FormInput.tsx

```diff
@@ -58,7 +61,7 @@ const FormInput: React.FC<FormInputProps> = ({
           ))}
         </Select>
       ) : (
-        <StyledInput defaultValue={defaultValue} id={id} {...rest} />
+        <StyledInput placeholder={placeholder} defaultValue={defaultValue} id={id} {...rest} />
```

Comment:
This `placeholder` property can be removed if we remove the placeholder prop from the parent.